GliaWidgets/Sources/EntryWidget/EntryWidget.swift

```diff
@@ -148,6 +148,7 @@ private extension EntryWidget {
         unreadSecureMessagesCount: Int?
     ) {
         if let ongoingEngagement {
+            environment.log.info("Preparing items based on ongoing engagement")
```

Comment:
We can benefit from adding test for this log - I think this is easy to do since logger is a dependency.
Also I'd like to ensure that this log was added to our logging events document to keep consistency with Android.